### DISCUSSION TITLE
Cannot click next to name of currently open view

### DISCUSSION THREAD

**Discussion by ClaudioFriederich** [community/community#123039] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-table-layout describes how to group/sort/filter etc. project table views.  All the procedures involve clicking the little down arrow next to the name of the currently open view, then selecting something from a context menu.  However, clicking the little down arrow just displays a tooltip saying "View Options" and does nothing.  It is thus effectively impossible to sort the view, for example.  Grouping is possible with the command palette, but sorting is not on the command palette.  Screen shot of what appears is below.  OS is Windows 2022 standard edition, browser is MS Edge, latest version as of 5/8/2022.
![GitHub Projects Bug](https://github.com/community/community/assets/57749337/de3efa1c-6e13-433f-a935-a55660ed8dc3)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ClaudioFriederich** [comment, marked as answer]
On further trying, it appears that https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-table-layout in GitHub documentation is inaccurate.  The proper procedure is to click the ... next to the field of interest, not the little down arrow next to the name of the currently open view the way the document explains.  GitHub works.  Documentation needs update.

**Reply by bohanwood** [reply]
This don't work in Board or Roadmap view.

**Comment by bohanwood** [comment]
This is definitely a bug. It has been working.
